src/Block/FeatureMediaBlockService.php

```diff
@@ -13,22 +13,64 @@
 
 namespace Sonata\MediaBundle\Block;
 
+use Sonata\AdminBundle\Admin\AdminInterface;
+use Sonata\AdminBundle\Form\Type\ModelListType;
+use Sonata\BlockBundle\Block\BlockContextInterface;
+use Sonata\BlockBundle\Block\Service\AbstractBlockService;
+use Sonata\BlockBundle\Block\Service\EditableBlockService;
 use Sonata\BlockBundle\Form\Mapper\FormMapper;
 use Sonata\BlockBundle\Meta\Metadata;
 use Sonata\BlockBundle\Meta\MetadataInterface;
 use Sonata\BlockBundle\Model\BlockInterface;
 use Sonata\Form\Type\ImmutableArrayType;
 use Sonata\Form\Validator\ErrorElement;
+use Sonata\MediaBundle\Model\MediaInterface;
+use Sonata\MediaBundle\Model\MediaManagerInterface;
+use Sonata\MediaBundle\Provider\Pool;
 use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
 use Symfony\Component\Form\Extension\Core\Type\TextareaType;
 use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\OptionsResolver\OptionsResolver;
+use Twig\Environment;
 
 /**
  * @author Thomas Rabaix <thomas.rabaix@sonata-project.org>
  */
-final class FeatureMediaBlockService extends MediaBlockService
+final class FeatureMediaBlockService extends AbstractBlockService implements EditableBlockService
 {
+    /**
+     * @var Pool
+     */
+    private $pool;
+
+    /**
+     * @var AdminInterface<MediaInterface>
+     */
+    private $mediaAdmin;
+
+    /**
+     * @var MediaManagerInterface
+     */
+    private $mediaManager;
+
+    /**
+     * @param AdminInterface<MediaInterface> $mediaAdmin
+     */
+    public function __construct(
+        Environment $twig,
+        Pool $pool,
+        AdminInterface $mediaAdmin,
+        MediaManagerInterface $mediaManager
+    ) {
+        parent::__construct($twig);
+
+        $this->pool = $pool;
+        $this->mediaAdmin = $mediaAdmin;
```

Comment:
This property make some issue. 
When we set `db_driver: 'no_driver'` then `sonata.media.admin.media` does not exists and throw exception. 
We should:
Allow create admin service with `manager_type: none` (or `manager_type: no_driver`).
Or make `mediaAdmin` optional - then also we should not allow generate edit form without `mediaAdmin`. 
Also if we do not have admin, then we should throw exception while generate edit form (is used by `EditableBlockService`). So can you change it to optional?

Reply:
Or we should find way to replace them.

Reply:
This is working exactly the same as the MediaBlockService.
The no_driver option is not suposed to be used by the users, it is just a temporary option.

If you want to improve that part. Please create another issue explaining why is that important and/or open a Pull request showing some code to implement that part.